Equilobe.DailyReport.SL/DbService.cs

```diff
@@ -55,5 +58,27 @@ public static string GetReportTime(string baseUrl, long projectId)
                 .Select(qr => qr.ReportTime)
                 .FirstOrDefault();
         }
+
+        public static void SetExecutionDate(long id)
```

Comment:
Metode de genul asta pot fi adaugate ca si extension methods pe ReportsDb. 

In servicii clasele intoarse de metode ar trebui sa fie diferite de clasele din Storage.